elfi/wrapper.py

```diff
@@ -6,6 +6,7 @@ class Wrapper():
     """ Wraps an external command to work as a callable operation for a node.
 
         Currently only supports sequential operations (not vectorized).
+        You can force this by setting batch_size=1 in the ABC method.
```

Comment:
Rewritten. As it is, sequential operations don't seem to work without setting batch_size=1.